tests/apidocs/openapi-derefed.json

```diff
@@ -4898,23 +4934,19 @@
                         "type": "object",
                         "required": [
                           "assets",
-                          "author",
```

Comment:
For this, I kept getting errors like `ValidationError: "'author' is a required property"` and it was present for _most_ of the entries in the `plugins` array but not all (like not `None` but non-existent`.